pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -53,6 +53,11 @@ const ProviderName = "openstack"
 var ErrNotFound = errors.New("Failed to find object")
 var ErrMultipleResults = errors.New("Multiple results where only one expected")
 var ErrNoAddressFound = errors.New("No address found for host")
+var ErrNoSubnetId = errors.New("subnet-id not set in cloud provider config")
```

Comment:
It's highly unlikely any caller is going to want to match against one of these different errors explicitly.
So instead of growing a huge list of pre-built errors here, delete these and just use in-line error strings in the function below:

```go
if len(lbOpts.SubnetId) == 0 {
      return fmt.Errorf("subnet-id not set in cloud provider config")
}
// etc
```